### PR TITLE
Use dropdown instead of radio buttons for selecting type of comment

### PR DESCRIPTION
**Pull request by RMacfarlane**
Some styling updates to the focused view, largely changing the control for choosing the comment type

### CODE REVIEW

File: webviews/components/comment.tsx [hunk 1]
```diff
@@ -318,100 +319,45 @@ export function AddComment({ pendingCommentText, state, hasWritePermission, isIs
...
-		selectedValue = e.target.value as ReviewType;
-		updatePR({ pendingReviewType: e.target.value as ReviewType });
-	}
-
-	return <form id='comment-form'
-		ref={form}
-		className='comment-form main-comment-form'
-		onSubmit={onSubmit}>
+	const availableActions = pr.isAuthor
+		? { 'comment': 'Comment '}
```
Comment (alexr00) [inline webviews/components/comment.tsx:352]: Why doesn't `{ 'comment': 'Comment '}` more closely match `COMMENT_METHODS`? I would expect it to be `{ comment: 'Comment'}` (no quotes and no space).

File: webviews/components/dropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+	const onMethodChange = e => {
+		selectMethod(e.target.value);
+		setOptionsVisible(false);
+		const primaryButton = document.getElementById('confirm-button');
+		primaryButton.focus();
+	}
+
+	const onKeyDown = e => {
+		if (e.keyCode === KEYCODES.esc && areOptionsVisible) {
```
Comment (alexr00) [inline webviews/components/dropdown.tsx:34]: Since this series of `if`s are mutually exclusive, it's slightly easier to read to make them `else if`s.

Comment (alexr00) [reply webviews/components/dropdown.tsx:34]: I see now that this a copy-paste. 🤷‍♀️

**Review by alexr00 — APPROVED** [review]

File: webviews/components/comment.tsx (hunk 1 above)
Comment (RMacfarlane) [reply webviews/components/comment.tsx:352]: Thanks, fixed!

File: webviews/components/dropdown.tsx (hunk 2 above)
Comment (RMacfarlane) [reply webviews/components/dropdown.tsx:34]: Restructured this to a `switch`, which I hope is easier to read

**Pull request merged**